Add quickstart for Configuration API (#743)
dapr/quickstarts: README.md

| [Secrets Management](./secrets_management/) | Securely fetch secrets |
| Actors | Coming soon... |

> | Observability | Coming soon... |

Reviewer: I never plan to have quickstart for observability or distributed lock.  Can i ask you to remove these Coming soons?  Actors ok to leave.

> | [Configuration](./configuration) | Consume configuration items as key/value pairs and subscribe to changes whenever a configuration item changes |

Reviewer 2: ```suggestion
| [Configuration](./configuration) | Get configuration items as key/value pairs or subscribe to changes whenever a configuration item changes |
```
